### DISCUSSION TITLE
Storing Images to AWS-S3 Failing due to java.io.IOException: Pipe closed

### DISCUSSION THREAD

**Discussion by RenukaPrasad-sai** [dcm4che/dcm4chee-arc-light#3710] [General]
**Describe the bug**
I configured dcm4chee-arc-ear-5.25.2-psql to store to the AWS S3 bucket. But C-Store is falling with "Caused by: java.io.IOException: Pipe closed" exception
command used to CStore : 

`storescu -c DCM4CHEE@localhost:11112 /c/RenukaTools/RenukaTools/SampleDICOMImages/Dog_15x15_40x.dcm
`

**To Reproduce**
1. Run dcm4chee-arc-ear-5.25.2-psql docker container using the docker-compose.yml
2. Setup AWS S3 following the instructions https://github.com/dcm4che/dcm4chee-arc-light/wiki/S3-Storage
3. Configure the Storage Descriptor for the DCM4CHEE device to use the AWS storage with the credentials downloaded from AWS  
4. Use storescu.bat command to store the images > 800 MB 
5. Store will fail with the above mentioned error

**Expected behavior**
Store should be successful and study can be viewed/exported without any issues

**Screenshots**
![image](https://user-images.githubusercontent.com/63426548/164504981-cc394e49-880d-4970-a0f1-b11c6172e767.png)

**Desktop (please complete the following information):**
 - OS: Linux Docker Engine created by Docker Desktop
 - Browser chrome
 - Version dcm4chee-arc-ear-5.25.2-psql

**Smartphone (please complete the following information):**

**Additional context**
Attaching the server.log from my testing 
[server.log](https://github.com/dcm4che/dcm4chee-arc-light/files/8532668/server.log)

**Comment by RenukaPrasad-sai** [comment]
I am having trouble in attaching the DICOM image used to upload. I did try at least 6 different images > 800 MB in size and all of them failed with the same error.
So I believe this issue can be recreated easily and consistently.

**Comment by gunterze** [comment]
> 	Caused by: org.jclouds.http.HttpResponseException: request: HEAD https://sais3awstest.s3-us-east-2.amazonaws.com/2022/04/21/31A1FF51/B9D23F31/42027F11 HTTP/1.1 failed with response: HTTP/1.1 503 Service Unavailable

Looks rather for an (configuration or runtime) error on AWS side.
